exporter/mezmoexporter/exporter_test.go

```diff
@@ -0,0 +1,143 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mezmoexporter
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/component/componenttest"
+	conventions "go.opentelemetry.io/collector/model/semconv/v1.6.1"
```

Comment:
Use `go.opentelemetry.io/collector/semconv`

Reply:
Corrected in [91b596c](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/9743/commits/91b596c92311ca12c487673070232591a198ca9d).